kubernetes/README.md

```diff
@@ -1,177 +1,10 @@
-* NodeNotReady
-* NodeoutofDisk
-* OutofDisk
-* Rebooted
-* TerminatedAllPods
-* Unable
-* Unhealthy
-
-### Service Checks
-The Kubernetes check does not include any service checks.
-
-## Troubleshooting
-### Can I install the agent on my Kubernetes master node(s) ?
-Yes, since Kubernetes 1.6, the concept of [Taints and tolerations][11] was introduced. Now rather than the master being off limits, it's simply tainted.  Add the required toleration to the pod to run it:
-
-Add the following lines to your Deployment (or Daemonset if you are running a multi-master setup):
-```
-spec:
- tolerations:
- - key: node-role.kubernetes.io/master
-   effect: NoSchedule
-```
-
-### Why is the Kubernetes check failing with a ConnectTimeout error to port 10250?
-The agent assumes that the kubelet API is available at the default gateway of the container. If that's not the case because you are using a software defined networks like Calico or Flannel, the agent needs to be specified using an environment variable:
-```
-          - name: KUBERNETES_KUBELET_HOST
-            valueFrom:
-              fieldRef:
-                fieldPath: spec.nodeName
-```
-See [this PR][12]
-
-###  Why is there a container in each Kubernetes pod with 0% CPU and minimal disk/ram?
-These are pause containers (docker_image:gcr.io/google_containers/pause.*) that K8s injects into every pod to keep it populated even if the "real" container is restarting/stopped.
-
-The docker_daemon check ignores them through a default exclusion list, but they will show up for K8s metrics like *kubernetes.cpu.usage.total* and *kubernetes.filesystem.usage*.
-
-## Further Reading
-
-To get a better idea of how (or why) to integrate your Kubernetes service, check out our [series of blog posts][13] about it.
-
-[1]: https://raw.githubusercontent.com/DataDog/integrations-core/master/kubernetes/images/kubernetes_dashboard.png
-[2]: https://github.com/DataDog/datadog-agent/blob/master/docs/agent/changes.md#kubernetes-support
-[3]: https://docs.datadoghq.com/agent/basic_agent_usage/kubernetes
-[4]: https://app.datadoghq.com/account/settings#agent
-[5]: https://github.com/DataDog/integrations-core/blob/master/kubernetes/datadog_checks/kubernetes/data/conf.yaml.example
-[6]: https://kubernetes.io/docs/api-reference/v1.7/#configmap-v1-core
-[7]: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account
-[8]: https://github.com/DataDog/integrations-core/blob/master/kubernetes/datadog_checks/kubernetes/data/conf.yaml.example#L118
-[9]: https://docs.datadoghq.com/agent/guide/agent-commands/?tab=agentv6#agent-status-and-information
-[10]: https://github.com/DataDog/integrations-core/blob/master/kubernetes/metadata.csv
-[11]: https://blog.kubernetes.io/2017/03/advanced-scheduling-in-kubernetes.html
-[12]: https://github.com/DataDog/dd-agent/pull/3051
-[13]: https://www.datadoghq.com/blog/monitoring-kubernetes-era
+Further documentation can be found in the Kubernetes stackpack
```

Comment:
remove the directory `kubernetes/images/kubernetes_dashboard.png`

Reply:
Done.